### DISCUSSION TITLE
How to keep it above table of contents

### DISCUSSION THREAD

**Discussion by george-gca** [francoischalifour/medium-zoom#207] [Q&A] [answered]
Currently we are using your zoom library in our [template site](https://github.com/alshedivat/al-folio), but when there is a table of contents, like in [this post](https://alshedivat.github.io/al-folio/blog/2023/sidebar-table-of-contents/), the toc keeps being drawn above the image.

![image](https://github.com/francoischalifour/medium-zoom/assets/31376482/be15ffad-f73d-4262-b579-2a8d3a96e547)

Any idea how to handle this case?

**Comment by francoischalifour** [comment, marked as answer]
You can specify the `z-index` of the Medium Zoom elements, as [explained in the documentation](https://github.com/francoischalifour/medium-zoom#the-zoomed-image-is-not-visible).

**Reply by george-gca** [reply]
Thanks for your help. In the end I not only had to set z-index of the image, but also the z-index of the other element to be smaller.
